src/Nordea/Components/DropdownOptions.elm

```diff
@@ -0,0 +1,252 @@
+                            , cursor pointer
+                            , hover [ backgroundColor Colors.coolGray ]
+                            , displayFlex
+                            , flexDirection row
+                            , padding (rem 0.5)
+                            , alignItems center
+                            , Css.property "gap" "0.5rem"
+                            ]
+                        ]
+                        [ Checkbox.init option.name (Html.text option.label) option.onCheck
+                            |> Checkbox.withIsChecked option.isChecked
+                            |> Checkbox.withAppearance Checkbox.Simple
+                            |> Checkbox.view []
+                        ]
+
+                dropdownStyles =
+                    Css.batch
+                        [ backgroundColor Colors.white
+                        , borderBottom3 (rem 0.0625) solid Colors.grayMedium
+                        , borderLeft3 (rem 0.0625) solid Colors.grayMedium
+                        , borderRight3 (rem 0.0625) solid Colors.grayMedium
+                        , borderBottomLeftRadius (rem 0.25)
+                        , borderBottomRightRadius (rem 0.25)
+                        , boxSizing borderBox
+                        , padding3 (rem 0.5) (rem 0) (rem 0.0)
+                        ]
+            in
+            Html.div
+                [ css
+                    [ displayFlex
+                    , flexDirection column
+                    , position absolute
+                    , top (pct 100)
+                    , left (rem 0)
+                    , right (rem 0)
+                    , if dropdown.hasFocus then
+                        display block
+
+                      else
+                        display none
+                    ]
+                ]
+                [ Html.ul
+                    [ css
+                        [ overflowY scroll
+                        , maxHeight (rem 16.75)
+                        , dropdownStyles
+                        , listStyle none
+                        ]
+                    ]
+                    (dropdown.options |> List.map viewOption)
+                ]
+    in
+    Html.column attrs
+        [ Html.row [ css [ displayFlex, justifyContent spaceBetween, marginBottom (rem 0.2) ] ]
+            [ Common.viewLabel { label = dropdown.label, isError = False } []
+            , dropdown.requirednessHint |> Html.viewMaybe RequirednessHint.view
+            ]
+        , Html.div
+            [ class "nordea-select"
+            , Events.on "focusout" (Decode.succeed (dropdown.onFocus False))
+            , Events.on "focusin" (Decode.succeed (dropdown.onFocus True))
+            , css [ position relative ]
+            , tabindex -1
+            ]
+            [ Html.select
+                [ css [ display none ] ]
+                (dropdown.options |> List.map (\option -> Html.option [ name option.name ] [ Html.text option.label ]))
```

Comment:
I wonder if a native accordion with a fieldset may better represent what we are doing here.

Check out this example https://jsfiddle.net/hvto259e/

This is natively supported, thus semantically correct, and does what we want out of the box. We only have to style it to look how we want

```html
<details>
  <summary>This is the collapsed label</summary>

  <fieldset>
    <label> 
        some label
        <input type="checkbox" />
    </label>
        <label> 
        some other label
        <input type="checkbox" />
    </label>
  </fieldset> 
</details>
```

Reply:
Good idea💡 It looks like the usage of `<summary>` is mainly intended for accordions with text. Let's keep what we have for now and we'll change it later if it is needed🙏

Reply:
Ah yes, that may be true. This seems like a good resource for making it accessible: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-multiselectable

Both the example with a ul and li (with aria props) and the one with label and select look very interesting!